doc/js-api/remotestorage.rst

```diff
@@ -76,6 +76,14 @@ List of events
 """""""""""""
    Emitted when a wire request completes
 
+``sync-req-done``
+"""""""""""""""""
+   Emitted when a single sync task has finished
```

Comment:
What does "sync task" mean exactly?

Reply:
If I understand correctly, a task can be a fetch of a directory or the upload of a local change.

Reply:
So why not say "sync request" then? The event's name already uses it and seems to be correct and descriptive in this case.